tephra-hbase-compat-0.96/src/test/java/co/cask/tephra/hbase96/coprocessor/TransactionProcessorTest.java

```diff
@@ -226,7 +226,11 @@ public void testDeleteFiltering() throws Exception {
       }
 
       // delete from the third entry back
-      Delete d = new Delete(row, V[5]);
+      long deleteTs = V[5] + 1;
```

Comment:
Since the data here was previously written, I don't think issuing a delete at the same timestamp makes sense.  That's not how a delete in a new transaction would work, which I think is what we are trying to emulate here.

In my opinion, the previous test code seemed off.